stringfunc.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import "strings"
+
+// StringFunc - string manipulation function wrappers
+type StringFunc struct {}
+
+func (t StringFunc) Replace(s, old, new string) string {
```

Comment:
> If necessary, a gomplate function named `replaceN` could be introduced calling `strings.Replace` directly.

To avoid the future name-shuffling (because I think exposing the full `strings.Replace` is inevitable) I think I'd prefer to name _this_ function `replaceAll`.

And as an aside, can you lower-case the `r` just so that this isn't public API right away? I feel a bit boxed in with all the exported funcs I've written to date, and I'd like to stop that and make new ones non-exported _for now_.